docs/environment-setup.md

```diff
@@ -168,6 +168,15 @@ _Note: If experiencing an error such as `Error starting userland proxy: listen t
 
 To run Elasticsearch natively, follow the instructions for your OS [here](https://www.elastic.co/guide/en/elastic-stack-get-started/7.6/get-started-elastic-stack.html#install-elasticsearch)
 
-### Finally
+### Finally:
 
+Telescope requires a running back-end and front-end to start.
+Start the backend for Telescope:
 Run `npm start` and open `localhost:3000`
```

Comment:
I think we can remove

> and open `localhost:3000`

since we're explaining later what to open depending on how the front-end will be run.